services/api-dotnet/README.md

```diff
@@ -1,50 +1,53 @@
-3. [Deployment automation](https://github.com/paralect/ship/tree/master/deploy/app) with Docker & Ansible. 
-4. [CI automation](https://github.com/paralect/ship/tree/master/deploy/drone-ci) using Drone CI (we might use other CI if this won't work)
-5. [.NET project srtart via single command](https://github.com/paralect/docker-compose-starter). We plan to use self-hosted API service to being able to run it within Docker. 
+# .NET API starter
+
+.NET is a huge part of the Paralect history. In fact, for the first 5 years of the company .NET was a primary technology for most of the projects. We love technology and beleive it suits very well for what we do: product engineering.
+
+This is a fully featured .NET restful API starter application, which is configured to work with either MongoDB or PostgreSQL. The goal of this project is to solve all routine tasks and keep your focus on the product and business logic of the application, not on the common things, such as logging, configuration, dev/production environments.
+
+## Projects
+
+|Name|Description|
+|:---|:----------|
+|[API (NoSQL version)](src/app/Api.NoSql)|.NET backend starter, configured to work with MongoDB|
+|[API (SQL version)](src/app/Api.Sql)|.NET backend starter, configured to work with PostgreSQL|
+|[API models](src/app/Api.Views)|The project to share models, mappings and validators across APIs|
+|[Common](src/app/Common)|DAL (for MongoDB and PostgresSQL) and some configuration, which can be shared across all applications|
+|[Services](src/app/Common.Services)|Domain/infrastructure services, shared across all applications|
+|[Scheduler](src/app/Scheduler)|Hangfire scheduler|
+|[Websocket server](src/app/SignalR)|SignalR websocket server (works with MongoDB only)|
+|[Unit tests for NoSQL API](src/app/Tests.NoSql)|Unit tests for NoSQL API|
+|[Unit tests for SQL API](src/app/Tests.Sql)|Unit tests for SQL API|
+
+## Features
+
+- configured console logging ([Serilog](https://serilog.net/))
+- request data validation ([FluentValidation](https://fluentvalidation.net/))
+- object mapping ([Automapper](https://automapper.org/))
+- email sender implementation for local development (can be used with [smtp4dev](https://github.com/rnwood/smtp4dev), for example)
+- Redis cache (see [CacheController](src/app/Api.NoSql/Controllers/CacheController.cs) for example of usage)
+- health checks
+- [Hangfire](https://www.hangfire.io/) scheduler. Can be configure to use either MongoDB or PostgreSQL storage. Dashboard is available on http://localhost:3001/hangfire
+- Feature flags support ([FeatureManagement](https://github.com/microsoft/FeatureManagement-Dotnet)). See [FeatureController](src/app/Api.NoSql/Controllers/FeatureController.cs) for example of usage)
+- Docker support
+- production ready account API resource (singup, signin, forgot password, reset password functionality)
```

Comment:
I'd move this point to the very top of the list as it's core of the project. Also believe it's beneficial to mention that API could be backed by Postgres and Mongo here too.

Reply:
Another point that could be mentioned in this list is DAL. We provide not only naked API but also a bunch of services for convenient data access. Such as filtration, pagination, sorting, etc.

Reply:
And last - we can also put here build-in Unit Testing feature.